main_test.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/yahoojapan/garm/config"
+)
+
+func TestParseParams(t *testing.T) {
+	type test struct {
+		name       string
+		beforeFunc func()
+		checkFunc  func(*params) error
+		checkErr   bool
+	}
+	tests := []test{
+		func() test {
+			return test{
+				name: "check parseParams set default value",
+				beforeFunc: func() {
+					os.Args = []string{""}
+				},
+				checkFunc: func(p *params) error {
+					if p.configFilePath != "/etc/garm/config.yaml" {
+						return errors.Errorf("unexpected file path. got: %s, want: /etc/athenz/client/config.yaml", p.configFilePath)
```

Comment:
Is this what you intend?
```suggestion
						return errors.Errorf("unexpected file path. got: %s, want: /etc/garm/config.yaml", p.configFilePath)
```

Reply:
yes, thx, fixed: 0f96865